### DISCUSSION TITLE
Zener diode temp coefficient

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#1082] [Q&A]
Hi,

I am trying to simulate the behaviour of a zener diode, BZT52C3V0.
It is specified to have a negative temperature coefficient (-3.5mV/C), but the voltage goes up with increasing temperature.
Is this normal?
I would expect the voltage to go down with increasing temperature.
Is there something wrong in the spice lib file?

[DIODES-BZT52CXVX.pdf](https://github.com/user-attachments/files/17846518/DIODES-BZT52CXVX.pdf)
[BZT52C3V0.spice.lib.zip](https://github.com/user-attachments/files/17846527/BZT52C3V0.spice.lib.zip)
[zenertest.sch.zip](https://github.com/user-attachments/files/17846531/zenertest.sch.zip)

**Comment by tomhajjar** [comment]
The Zener diode model is deficient. I found a similar diode that has a more complete model.

On a side note: The term "Zener" diode is ONLY valid for diodes say 5 Volts or less. Diodes > 5 Volts are actually Avalanche diodes. You can tell by the "knee" of the breakover region and the Tempco. Zener diodes have a "soft" knee while Avalanche diodes have a "sharp" knee.

Manufacturers used to sell "Voltage Reference" diodes that consisted of multiple dies in series in a package to achieve specific a voltage and Tempco. IC voltage references killed that market.

![2024-11-21_143116](https://github.com/user-attachments/assets/9d3db92c-b7fa-4380-9d24-e3d6911ee3cd)

[Zener_Diode_tempco_prj.zip](https://github.com/user-attachments/files/17851015/Zener_Diode_tempco_prj.zip)

**Comment by bnilsson11** [comment]
Thanks, it seems the models from Diodes Inc. are useless in the tempco context.
If I understand correctly, MMSZ5225BT1G is a Onsemi device .
Where did you find this model? 
How "generic" is this kind of spice model?

**Comment by tomhajjar** [comment]
Search OnSemi for the model.

Tempco is meaningless for most uses of Zener diodes. That is especially so for ones with soft knees. The days of using precision Voltage Reference diodes ended decades ago. They are still used in legacy designs especially for the Military. Some would be used in ovens like a crystal to make a stable reference. ICs came out in the 1980's that were better.

https://www.raffia.ch/content/datasheets/volume03/1N823_AppNotes_Motorola.pdf
https://ww1.microchip.com/downloads/aemDocuments/documents/HRDS/ProductDocuments/DataSheets/1N821-1N829A-1.pdf
